ext/linux/include/lk_atomics.h

```diff
@@ -251,7 +251,9 @@ static inline uint16_t xchg_release16(uint16_t *ptr, uint16_t val) {
 static inline void cpu_relax (void) {
 #if defined(__x86_64__)
 	asm volatile ("pause" : : : "memory" );
-#elif defined (__arch64__)
+#elif defined (__aarch64__) && defined(RELAX_IS_ISB)
```

Comment:
When do you recommend a user enabling this over the default? Can you add a define in the Makefile?

Reply:
Without the define, this behaves like the kernel does today. I believe the define improves the performance of the kernel locks but it's not what is committed today.

Reply:
Ok so if the kernel changes in the future we just need to expose this define.